### PR TITLE
AFT Updates

### PR DESCRIPTION
**Pull request by xw-g**

### CODE REVIEW

File: feature/gribi/encap_frr/README.md [hunk 1]
```diff
@@ -257,6 +257,7 @@ network-instances {
 IPv4Entry {138.0.11.0/24 (ENCAP_TE_VRF_A)} -> NHG#10 (DEFAULT VRF) -> {
   {NH#201, DEFAULT VRF, weight:1},
   {NH#202, DEFAULT VRF, weight:3},
+  backup_next_hop_group: 100 // in case specific vendor implementation or bugs pruned the NHs.
```
Comment (nachikethas) [inline feature/gribi/encap_frr/README.md:260]: Should this say 200?

File: feature/gribi/encap_frr/README.md [hunk 2]
```diff
@@ -428,16 +438,29 @@ the double failure handling, and ensures that the fallback to DEFAULT is
...
 
-1.  Update `NHG#8` and `NHG#9` to the following: ``` NHG#8 (Default VRF) {
-    {NH#1000, DEFAULT VRF} } NH#1000 ->
-    { decapsulate_header: OPENCONFIGAFTTYPESDECAPSULATIONHEADERTYPE_IPV4 network_instance: "DEFAULT" }
+1.  Update `NHG#8` and `NHG#9` to the following:
 
-NHG#9 (Default VRF) { {NH#1001, DEFAULT VRF} } NH#1001 -> { decapsulate_header:
-OPENCONFIGAFTTYPESDECAPSULATIONHEADERTYPE_IPV4 network_instance: "DEFAULT" } ```
+    ```
+    NHG#100 (Default VRF) {
```
Comment (nachikethas) [inline feature/gribi/encap_frr/README.md:444]: NHG 100, and NHG 101 are pretty munch identical. Is there any reason to keep them separate?

File: feature/gribi/encap_frr/README.md [hunk 3]
```diff
@@ -275,11 +276,18 @@ NH#202 -> {
   network_instance: "TE_VRF_111"
 }
 
+NHG#200 (Default VRF) {
+  {NH#2000, DEFAULT VRF, weight:1}
+}
+NH#2000 -> {
+    decapsulate_header: OPENCONFIGAFTTYPESDECAPSULATIONHEADERTYPE_IPV4
```
Comment (nachikethas) [inline feature/gribi/encap_frr/README.md:283]: We shouldn't need to decap here right?

File: feature/gribi/encap_frr/README.md (hunk 2 above)
Comment (xw-g) [reply feature/gribi/encap_frr/README.md:443]: they "were" different (encap to different tunnels). To keep things simple, I just do the inline update here.

File: feature/gribi/encap_frr/README.md (hunk 3 above)
Comment (xw-g) [reply feature/gribi/encap_frr/README.md]: sorry, pasted the wrong changes. fixed now.

**Comment by xw-g** [comment]
Approval received from internal b/304783132.

**Pull request merged**